include/zephyr/net/dhcpv4.h

```diff
@@ -163,6 +163,48 @@ int net_dhcpv4_remove_option_callback(struct net_dhcpv4_option_callback *cb);
 
```

Comment:
BTW, just noticed this. Could you use lower case letters in commit subject, so `net: dhcpv4: ....`, same for the testing commit, which should be called `tests: net: dhcpv4: ....`